'operator_usage_whitespace' rule conflicts with Image Literal

When using image literal, It's should not triggered

I'm sorry I can't reproduce this with the code you've shared.

```
$ swiftlint version
0.14.0
$ cat imageliteral.swift
imageView.image = #imageLiteral(resourceName: "image")   // Triggered

button.setImage(#imageLiteral(resourceName: "image"), for: .normal)   // Triggered
button.setImage(#imageLiteral(resourceName: "image") , for: .normal)   // Triggered
button.setImage( #imageLiteral(resourceName: "image"), for: .normal)   // Triggered
button.setImage( #imageLiteral(resourceName: "image") , for: .normal)   // Triggered

let imageView = UIImageView(image: #imageLiteral(resourceName: "image"))   // Triggered
let imageView = UIImageView(image: #imageLiteral(resourceName: "image") )   // Triggered

let images = [ #imageLiteral(resourceName: "image-1"), #imageLiteral(resourceName: "image-2") ]   // Triggered

let a  = 0 // I added this to show the rule is working...
$ swiftlint lint --path imageliteral.swift
Loading configuration from '.swiftlint.yml'
Linting Swift files at path imageliteral.swift
Linting 'imageliteral.swift' (1/1)
imageliteral.swift:13:6: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
Done linting! Found 1 violation, 0 serious in 1 file.
```

```
$ swiftlint version
0.14.0

$ cat .swiftlint.yml
whitelist_rules:
  - operator_usage_whitespace

$ cat test.swift 
imageView.image = #imageLiteral(resourceName: "image-1")   // Triggered

button.setImage(#imageLiteral(resourceName: "image-1"), for: .normal)   // Triggered
button.setImage(#imageLiteral(resourceName: "image-1") , for: .normal)   // Triggered
button.setImage( #imageLiteral(resourceName: "image-1"), for: .normal)   // Triggered
button.setImage( #imageLiteral(resourceName: "image-1") , for: .normal)   // Triggered

let imageView = UIImageView(image: #imageLiteral(resourceName: "image-1"))   // Triggered
let imageView = UIImageView(image: #imageLiteral(resourceName: "image-1") )   // Triggered

let images = [ #imageLiteral(resourceName: "image-1"), #imageLiteral(resourceName: "image-2") ]   // Triggered

$ swiftlint lint --path test.swift
Loading configuration from '.swiftlint.yml'
Linting Swift files at path test.swift
Linting 'test.swift' (1/1)
test.swift:1:53: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:3:51: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:4:51: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:5:52: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:6:52: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:8:70: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:9:70: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:11:50: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
test.swift:11:90: warning: Operator Usage Whitespace Violation: Operators should be surrounded by a single whitespace when they are being used. (operator_usage_whitespace)
Done linting! Found 9 violations, 0 serious in 1 file.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -57,6 +57,11 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1026](https://github.com/realm/SwiftLint/issues/1026)
 
+* Fix false positive in `operator_usage_whitespace` rule when
+  using image literals.  
+  [Marcelo Fabri](https://github.com/marcelofabri)
+  [#1028](https://github.com/realm/SwiftLint/issues/1028)
+
 ## 0.14.0: Super Awesome Retractable Drying Rack
 
 ##### Breaking

--- a/Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift
+++ b/Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift
@@ -32,7 +32,9 @@ public struct OperatorUsageWhitespaceRule: OptInRule, CorrectableRule, Configura
             "let range = 1 ... 3\n",
             "let range = 1..<3\n",
             "#if swift(>=3.0)\n",
-            "array.removeAtIndex(-200)\n"
+            "array.removeAtIndex(-200)\n",
+            "let name = \"image-1\"\n",
+            "button.setImage(#imageLiteral(resourceName: \"image-1\"), for: .normal)\n"
         ],
         triggeringExamples: [
             "let foo = 1↓+2\n",
@@ -105,7 +107,7 @@ public struct OperatorUsageWhitespaceRule: OptInRule, CorrectableRule, Configura
             zeroSpaces + trailingVariableOrNumber
         let excludingPattern = "(?:\(genericPattern)|\(validRangePattern))"
 
-        let kinds = SyntaxKind.commentAndStringKinds()
+        let kinds = SyntaxKind.commentAndStringKinds() + [.objectLiteral]
 
         return file.matchPattern(pattern, excludingSyntaxKinds: kinds,
                                  excludingPattern: excludingPattern).flatMap {

